Update docker commands in Lisk Service docs v0.7.0 (#1722)
LiskHQ/lisk-service: docs/antora/modules/ROOT/pages/management/docker.adoc

$ docker exec -it lisk-service_mysql_core_1 bash # <1>
----

<1> Connect to the interactive shell of the MySQL Docker container in which you wish to reset the MySQL database.

=== Drop Database

[tabs]
====
MySQL DB::
+
--
[source,bash]
----

> root@f1413b535254:/# mysql -ulisk -ppassword # <1>

Reviewer: ```suggestion
root@f1413b535254:/# mysql -ulisk -ppassword
```

please remove the numbers in the cod snippet, because they are currently not rendered correctly in the docs.

Author: OK, I have removed all the numbers from both code snippets 👍

mysql> create database lisk; # <3>
mysql> quit; # <4>
----
<1> Login to MySQL with the `lisk` user.
<2> Drop the `lisk` database.
<3> Create a fresh database `lisk`.
<4> Log out from MySQL by typing `quit;`.
Log out of the docker container by pressing kbd:[CRTL] + kbd:[D].

--

> Maria DB::

Reviewer 2: ```suggestion
MariaDB::
```

+
--
The Maria DB also uses the same SQL commands for querying and manipulating the relevant data, including data storage, client libraries and replication.

[source,bash]
----

> docker exec -it lisk_mariadb mysql -u root -p # <1>
> drop database lisk; # <2>
> create database lisk; # <3>
> exit; # <4>

Reviewer 2: Please feel free to cleanup the intermediate logs, if necessary.
```suggestion
sameer@Sameers-MBP mariadb % docker exec -it lisk_mariadb mysql -u root -p # <1>
Enter password: 
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 9
Server version: 10.7.8-MariaDB-1:10.7.8+maria~ubu2004 mariadb.org binary distribution

Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MariaDB [(none)]> drop database lisk; # <2>
Query OK, 0 rows affected (0.009 sec)

MariaDB [(none)]> create database lisk; # <3>
Query OK, 1 row affected (0.001 sec)

MariaDB [(none)]> exit; # <4>
Bye
```

or 

```suggestion
sameer@Sameers-MBP mariadb % docker exec -it mariadb-mariadb-1 mysql -uroot -ppassword # <1>
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 21
Server version: 10.7.8-MariaDB-1:10.7.8+maria~ubu2004 mariadb.org binary distribution

Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MariaDB [(none)]>  drop database lisk; # <2>
Query OK, 0 rows affected (0.009 sec)

MariaDB [(none)]> create database lisk; # <3>
Query OK, 1 row affected (0.001 sec)

MariaDB [(none)]> exit; # <4>
Bye
```

mysql> create database lisk; # <3>
mysql> quit; # <4>
----
<1> Login to MySQL with the `lisk` user.
<2> Drop the `lisk` database.
<3> Create a fresh database `lisk`.
<4> Log out from MySQL by typing `quit;`.
Log out of the docker container by pressing kbd:[CRTL] + kbd:[D].

--
MariaDB::
+
--

> The Maria DB also uses the same SQL commands for querying and manipulating the relevant data, including data storage, client libraries and replication.

Reviewer 2: ```suggestion
MariaDB also uses the same SQL commands for querying and manipulating the relevant data, including data storage, client libraries and replication.
```

Author: Done!
